### PR TITLE
Hi Jeff, I added a couple names to the list.

### PR DESCRIPTION
**Pull request by birdsandbugs**
@jeffhoul please consider these changes (if you think they could be improvements)!

Best,

Graham

### CODE REVIEW

**Comment by jeffhoul** [comment]
That's great - let's do that, Graham

File: README.md [hunk 1]
```diff
@@ -11,4 +11,5 @@ Here is a list:
 * Yvan Cournoyer
 * Booby Orr
 * Jacques Lemaire
+* Captain Kirk
```
Comment (birdsandbugs) [inline README.md:14]: yadda yadda yadda.

**Comment by jeffhoul** [comment]
It's all in

**Comment by birdsandbugs** [comment]
Fantastic 💯

**Pull request merged**